src/vs/base/common/extpath.ts

```diff
@@ -139,8 +139,8 @@ export function isUNC(path: string): boolean {
 }
 
 // Reference: https://en.wikipedia.org/wiki/Filename
-const WINDOWS_INVALID_FILE_CHARS = /[\\/:\*\?"<>\|]/g;
-const UNIX_INVALID_FILE_CHARS = /[\\/]/g;
+const WINDOWS_INVALID_FILE_CHARS = /\.\.\/|\\|(?<!\.)\/|:|\*|\?|"|<|>|\|/g;
```

Comment:
This regexes grew a lot. Is it possible to have this in simpler form?
Or to fix it some other way?

Reply:
Do you mean Windows regex only or both of them?

Reply:
Both trippled in size.

Reply:
@isidorn I`ve managed to shorten Windows regex, but there is no way to reduce Unix regex. Do you think that separate if statement will do better?